examples/erc721/tests/erc721.rs

```diff
@@ -1392,6 +1408,24 @@ async fn unpauses(alice: Account) -> eyre::Result<()> {
     Ok(())
 }
 
+#[e2e::test]
+async fn unpause_reverts_in_unpaused_state(alice: Account) -> eyre::Result<()> {
```

Comment:
Are we switching to `<func_name>_reverts_<description>` convention already?

Reply:
Make sense to change it everywhere

Reply:
Yes, let's start adding tests that follow this naming convention. We may rename the rest in the future.